pkg/actions/iamidentitymapping/create.go

```diff
@@ -0,0 +1,78 @@
+package iamidentitymapping
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
```

Comment:
we've moved to aws-sdk-go-v2 now, can you use v2 here?

Reply:
Will update it, thanks!